ng-dev/utils/git/github-urls.ts

```diff
@@ -31,7 +37,7 @@ export function getRepositoryGitUrl(
   githubToken?: string,
 ): string {
   if (config.useSsh) {
-    return `git@github.com:${config.owner}/${config.name}.git`;
+    return `${GITHUB_SSH_PREFIX}:${config.owner}/${config.name}.git`;
   }
   const baseHttpUrl = `https://github.com/${config.owner}/${config.name}.git`;
```

Comment:
Nit: you could potentially use the variable you introduced here and below as well, but not sure if it's even worth maintaing the variable. If you want to keep it though, it would make sense to use it consistenly I think